### PR TITLE
refactor: Rewrite BehaviorMessage classes to use member initialization, preferred member naming conventions, and const-ref getters

### PR DESCRIPTION
**Pull request by jadebenn**
Changes split out from #1452 for more focused PR.

### CODE REVIEW

**Review by jadebenn — COMMENTED** [review]
This leaves out the unique pointers, use of string_views, and removal of raw pointers in favor of references featured in #1452 in order to minimize the (unfortunately still large) amount of code touched. I will probably split each of those aforementioned points into individual PRs as well after this one is or isn't approved.

**Comment by jadebenn** [comment]
Whoops, sorry about the inadvertent assignment there. Wrong button.

**Review by aronwk-aaron — DISMISSED** [review]

File: dGame/dPropertyBehaviors/ControlBehaviorMessages/AddStripMessage.cpp [hunk 1]
```diff
@@ -2,27 +2,24 @@
...
+	const auto* const actions = strip->GetArray("actions");
 	if (!actions) return;
 
 	for (uint32_t actionNumber = 0; actionNumber < actions->GetDense().size(); actionNumber++) {
-		auto* actionValue = actions->GetArray(actionNumber);
+		const auto* const actionValue = actions->GetArray(actionNumber);
 		if (!actionValue) continue;
 
-		actionsToAdd.push_back(Action(actionValue));
+		m_ActionsToAdd.push_back(Action{ actionValue });
```
Comment (jadebenn) [inline dGame/dPropertyBehaviors/ControlBehaviorMessages/AddStripMessage.cpp:20]: changing this to be an emplace

**Review by aronwk-aaron — APPROVED** [review]

**Comment by jadebenn** [comment]
I understand if there's reluctance to go through this because of its size, but this was about the most focused I could make these changes. It's just three things (member naming, ordering, and functions) altered over that amount of files.

**Review by EmosewaMC — APPROVED** [review]

**Pull request merged**